plugins/module_utils/copy.py

```diff
@@ -108,12 +121,66 @@ def copy_pds2uss(src, dest):
     src = _validate_data_set_name(src)
     dest = _validate_path(dest)
     cp_pds2uss = "cp -U -F rec \"//'{0}'\" {1}".format(src, quote(dest))
+    if is_binary:
+        cp_pds2uss = cp_pds2uss.replace("rec", "bin")
```

Comment:
Since Python's string replacement method `replace()` replaces all occurrences by default, I think this may result in unexpected behavior when the *src* or *dest* contain the string "rec". Would it be possible do to one of the following, or something different if you prefer:

1. Add the count argument of `1` to each call to `replace()`, probably the easiest change: [more info here](https://docs.python.org/3/library/stdtypes.html#str.replace)
2. update the strings to take another argument to determine if `rec` or `bin` should be used, basic example:
```python
copy_type = "bin" if is_binary else "rec"
cp_uss2mvs = "cp -CM -F {0} {1} \"//'{1}'\"".format(copy_type, quote(src), dest)
```
I personally find second method a bit cleaner but that is just personal preference